ocpmodels/trainers/forces_trainer.py

```diff
@@ -612,8 +612,14 @@ def run_relaxations(self, split="val", epoch=None):
                     gather_results["pos"].extend(rank_results["pos"])
                     os.remove(rank_path)
 
+                # Because of how distributed sampler works, some system ids
+                # might be repeated to make no. of samples even across GPUs.
```

Comment:
Thinking about this some more...don't we have this issue in all the other save prediction calls as well then?

Reply:
Probably not because we don't use `DistributedSampler`?

Reply:
We do in the energy trainer https://github.com/Open-Catalyst-Project/ocp/blob/911ee9e9130cdef4e69b06afacea64902f233b41/ocpmodels/trainers/energy_trainer.py#L128-L159

Reply:
Good catch! Could you check if the latest commit fixes it?

Reply:
Yup LGTM - thanks!